### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by ADmad**
Having the action buttons inline with header was silly IMHO. It meant it's location always kept changing based on header length.

### CODE REVIEW

File: src/Template/Element/action-header.ctp [hunk 1]
```diff
@@ -8,4 +8,5 @@ if (!$this->exists('actions')) {
     $this->end();
 }
 ?>
-<h2><?= $this->get('title'); ?><span class="actions"><?= $this->fetch('actions'); ?></span></h2>
```
Comment (jippi) [inline src/Template/Element/action-header.ctp]: or right-align them?

Comment (ADmad) [inline src/Template/Element/action-header.ctp]: What if a record has a long ass title and one is checking on a smaller screen resolution?

File: src/Template/Layout/default.ctp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="<?= str_replace('_', '-', \Cake\I18n\I18n::locale()) ?>">
```
Comment (lorenzo) [inline src/Template/Layout/default.ctp]: Use this funiiton instead

Comment (ADmad) [inline src/Template/Layout/default.ctp]: Why get fancy when a simple string replace works :stuck_out_tongue:

**Comment by ADmad** [comment]
@lorenzo Done

**Pull request merged**